packages/formspree-core/src/forms.ts

```diff
@@ -1,6 +1,6 @@
 import type { PaymentMethodResult } from '@stripe/stripe-js';
 
-export type SubmissionData = FormData | any;
+export type SubmissionData = FormData | Record<string, string | Blob>;
```

Comment:
`string | Blob` matches the accepted types of `FormData` `append`, `set` methods. https://developer.mozilla.org/en-US/docs/Web/API/FormData/append